src/Orchard.Cms.Web/Startup.cs

```diff
@@ -53,8 +53,11 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseStaticFiles();
 
-            loggerFactory.AddConsole(Configuration);
-            loggerFactory.AddDebug();
+            if (env.IsDevelopment())
```

Comment:
Only the `Debug` should be in Development, the prod might use the console. And the console can be driven by configuration, unlike debug.